### ISSUE TITLE
Change npm ownership

### ISSUE THREAD

**Issue by geoidesic** [reststate/reststate-vuex#283]
Hi @CodingItWrong could you please give me ownership? I'd be happy to take this over.

If you could give me ownership over the reststate GitHub account and the npm account I'd be happy to take over, maintain and grow this library. I think that may be better for existing users than me creating a fork that then has to have a new npm registry account etc.

Please let me have your thoughts.

**Comment by CodingItWrong** [comment]
I'm not comfortable passing maintainership of this library to someone else, because I want to ensure users make their own decisions about whether to trust a new maintainer.

But since you have a solid track record using the library, if you publish a fork I would be comfortable linking to it at the top of the readme.

**Comment by issa** [comment]
May I enquire why you decided to no longer maintain it? Just because of time and resources? Or are there some fundamental problems with the library?

**Comment by CodingItWrong** [comment]
@issa Yes, because of time and resources, related to the fact that I'm no longer actively developing in Vue, so not getting a chance to use the library actively. No real fundamental problems with the library, no.

**Comment by lucianholt97** [comment]
There's also a typed alternative which is worth looking at:
https://www.npmjs.com/package/@anny.co/vue-jsonapi-orm

**Comment by freelock** [comment]
Just stumbled across this library, which seems to be such an elegant solution compared to the alternatives. Sad to see it marked unmaintained -- @geoidesic are you planning to pick this up?

**Comment by geoidesic** [comment]
> I'm not comfortable passing maintainership of this library to someone else, because I want to ensure users make their own decisions about whether to trust a new maintainer.
> 
> But since you have a solid track record using the library, if you publish a fork I would be comfortable linking to it at the top of the readme.

Ok thanks. I'll do that then. What do you suggest wrt npm? Most people will install this via npm and currently that will install yours. Should I start a new npm repo for my fork?

**Comment by CodingItWrong** [comment]
Yes, I'm recommending both a separate npm package name and a separate GitHub repo.
